setup.py

```diff
@@ -52,6 +53,17 @@ def read_file(filename, encoding='utf8'):
     'zope.interface',
 ]
 
+# Add pywin32 on Windows platforms to handle low-level system calls.
+# This dependency needs to be added using environment markers to avoid its installation on Linux.
+# However environment markers are supported only with setuptools >= 36.2.
+# So this dependency is not added for old Linux distributions with old setuptools,
+# in order to allow these systems to build certbot from sources.
+if StrictVersion(setuptools_version) >= StrictVersion('36.2'):
+    install_requires.append('pywin32 ; sys_platform == \'win32\'')
```

Comment:
style nit: might as well use double outer quotes to avoid having to backslash.

Reply:
I do not know why, I hate double quotes, pushing me to use less-optimal solutions to avoid them ...

So, fixed.